pkg/cmd/extension/manager.go

```diff
@@ -591,8 +597,155 @@ func (m *Manager) Create(name string) error {
 	}
 	dir := filepath.Join(wd, name)
 	addCmd := m.newCommand(exe, "-C", dir, "--git-dir="+filepath.Join(dir, ".git"), "add", name, "--chmod=+x")
+	return addCmd.Run()
+}
+
+func (m *Manager) otherBinScaffolding(gitExe, name string) error {
+	err := os.MkdirAll(filepath.Join(name, ".github", "workflows"), 0755)
+	if err != nil {
+		return err
+	}
+	workflow := heredoc.Doc(`
+	name: release
+	on:
+	  push:
+	    tags:
+	      - "v*"
+	permissions:
+	  contents: write
+	
+	jobs:
+	  release:
+	    runs-on: ubuntu-latest
+	    steps:
+	      - uses: actions/checkout@v2
+	      - uses: cli/gh-extension-precompile@v1
+		  	with:
+			  build_script_override: "script/build.sh"
+	`)
+	workflowPath := filepath.Join(".github", "workflows", "release.yml")
+	err = ioutil.WriteFile(filepath.Join(name, workflowPath), []byte(workflow), 0755)
+	if err != nil {
+		return err
+	}
+	err = os.Mkdir(filepath.Join(name, "script"), 0755)
+	if err != nil {
+		return err
+	}
+	buildScript := heredoc.Doc(`
+	#!/usr/bin/env bash
+	echo "TODO implement this script."
+	echo "It should build binaries in /dist/<platform>-<arch>[.exe] as needed."
```

Comment:
This example script could exit with a nonzero status so that if the user forgets to write it, it will obviously fail in CI when building the binaries.